pages/windows/assoc.md

```diff
@@ -14,3 +14,7 @@
 - Set the associated file type for a specific extension:
 
 `assoc .{{txt}}={{txtfile}}`
+
+- View the output of assoc one screen at a time:
+
+`assoc | more`
```

Comment:
```suggestion
- View the output of `assoc` one screen at a time:

`assoc | {{more}}`
```

- Optional fix: command names in description can be quoted.
- Optional fix: `more` can be enclosed in placeholder as `assoc` doesn't require just `more` command to be on the right side.
- Warning: too many `more`-like commands can be substituted. For that reason all examples conforming the following pattern: `{{command_from_page_name}} | {{page}}` can be removed everywhere.